docs/DeveloperGuide.md

```diff
@@ -31,4 +174,27 @@
 
 ## Instructions for manual testing
```

Comment:
Is it better to follow the format given in [here](https://se-education.org/addressbook-level3/DeveloperGuide.html#appendix-instructions-for-manual-testing)?